### PR TITLE
Complete mobile version

### PR DESCRIPTION
**Pull request by DevKaranJ**
Completed mobile version of portfolio website section. Implemented the 'Works' section, 'About myself' section, and toolbar. Used CSS Grid and Flexbox for layout. Added ul and li elements for categories in the 'Works' and 'About myself' sections.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -35,5 +35,188 @@ <h1>I’m Karan.
...
+                           <li><a href="https://www.angellist.com/"><img src="./resource/angelist.svg" alt="Angelist icon" /></a></li>
+                        </ul>
+                        </div>
+                <div class="resumeButton">
+                    <a href="./resource/Resume.pdf" target="_blank" rel="resume">
+                    <input class="resumeButton" type="button" value="Get my resume">
+                </div>
+            </div>
+            </div>
+            <section class="techSkills">
```
Comment (topeogunleye) [inline index.html:177]: - [ ] I kindly suggest that you also use a `ul` for your `techSkills`, and create and `li` tag container for its children.

## EXAMPLE
```
   <ul>
        <li class="languague">
              <ul>
                  <li><a href="#">JavaScript</a></li>
                  <li><a href="#">HTML</a>
                  <li><a href="#">CSS</a>
             </ul>
         </li>
          <li class="framework">
              <ul>
                  <li><a href="#">Bootstrap</a></li>
                  <li><a href="#">Ruby on Rails</a>
                  <li><a href="#">RSPEC</a>
             </ul>
         </li>
            <li class="skills">
              <ul>
        
          <li><a href="#">Code Kit</a></li>
                  <li><a href="#">GitHub</a>
                  <li><a href="#">Codepen</a>
             </ul>
         </li>                                     
</ul>```

Please note that the above is just an example and may not apply directly to your code.

File: index.html [hunk 2]
```diff
@@ -35,5 +35,188 @@ <h1>I’m Karan.
                     </ul>
                     </div>
         </section>
+        <!--the work section-->
+        <section class="work">
+            <!--first card-->
+            <div class="card">
```
Comment (topeogunleye) [inline index.html:41]: - [ ] I kindly suggest that you reduce the margin (spacing outside the project cards) to make your portfolio match the design more closely:

| Your Design  | Figma Design |
| ------------- | ------------- |
| image  | image  |

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @DevKaranJ, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 

- The container of the section is a section tag 📦
- The section has the proper background image 🌅
- The title uses a header tag hx 📝
- The content of each project is aligned with flexbox 🤸‍♀️
- The project title uses a header tag hx 📝
- The project has an image 🖼️
- The paragraph is a p tag 📜
- The button is a link a or button tag, and follows the button styling: Active, Hover and Pressed 🎨
- The list of project technologies are a list ul and each of them a li 📋

And for the About myself section:

- The container of the section is a section tag 📦
- The section has the proper background image 🌅
- The title uses a header tag hx 📝
- The paragraph is a p tag 📜
- other titles use a header tag hx 📝
- The list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag) 🔗
- The button is a link a or button tag, and follows the button styling: Active, Hover and Pressed 🎨

Keep up the great work! 👍

I hope that helps!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Cabraham1 — APPROVED** [review]
Hi @DevKaranJ  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**